repos.txt

```diff
@@ -77,6 +82,7 @@ java-network-management
 java-network-security
 java-networkconnectivity
 java-notebooks
+java-notification
```

Comment:
While this is a handwritten library, as long as there're no dedeicated team to maintain the repositories, we maintain it. We should add it to monorepo for the sake of repository maintenance.